assembl/static2/tests/unit/components/cookiesBar.spec.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import { DumbCookiesBar } from '../../../js/app/components/cookiesBar';
+
+describe('CookiesBar component', () => {
+  it('should render a bar with 2 buttons to accept the cookies policy or read them (bar is hidden by default)', () => {
+    const component = renderer.create(<DumbCookiesBar />);
```

Comment:
your component wants those props: `client`, `acceptedCookies` (or maybe the mutation and `acceptedCookies` if you don't use `client` anymore (see my comment above))